scripts/release/deploy_staging_jars.sh

```diff
@@ -100,15 +104,17 @@ fi
 COMMON_OPTIONS="-DdeployArtifacts=true -DskipTests -DretryFailedDeploymentCount=10"
 for v in "${ALL_VERSION_OPTS[@]}"
 do
-  # clean everything before any round of depoyment
-  $MVN clean $COMMON_OPTIONS
```

Comment:
wouldn't this clean everything?

Reply:
We have to clean every module to give a clean start for each profile.  `mvn install` below has already built everything and we don't want the artifacts from previous round to interfere with this round.